go.mod

```diff
@@ -43,16 +43,22 @@ require (
 	k8s.io/client-go v0.28.4
 	k8s.io/kubectl v0.28.4
 	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
+	kmodules.xyz/client-go v0.25.41
```

Comment:
This package has a name that it's easy to get confused.

Reply:
bypassed with local change